frontend/src/components/Skeletons/SkeletonItemGrid.vue

```diff
@@ -15,11 +15,8 @@ import { computed } from 'vue';
 import { CardShapes, getShapeFromItemType } from '@/utils/items';
 import { useResponsiveClasses } from '@/composables';
 
-const props = defineProps({
-  viewType: {
-    type: String,
-    default: (): string => 'Movie'
-  }
+const props = withDefaults(defineProps<{ viewType?: string }>(), {
```

Comment:
Looking at GitHub, so maybe I'm not recalling stuff correctly, but, afaik, the viewTypes are BaseItemKind, right? Doesn't make sense to use the BaseItemKind type?

Reply:
I didn't look at this when I converted it initially since I was scoping my changes, but I think you're right, and there's very few uses of this component. I'll add a commit for this change 🚀

Reply:
Update from some quick investigation: `library/_viewId` will need some large refactors, so I'll look into redoing it with composition API but that will take a while. I'll work on it but I don't think we need to block this PR on that being done.

Reply:
@aweebs Yup of course. In fact, I wouldn't worry too much about migrating that page to Composition API for the moment. Look at this: https://github.com/jellyfin/jellyfin-vue/discussions/1679

In order to accomplish that new style of media-type centric navigation, new library pages for every kind of type (artists, genres, shows, etc) will be created, so the logic of each page will be more simple. It's probably not worth the effort to port anything when we want to rewrite that entirely in the near future.